core-bundle/src/Resources/config/services.yml

```diff
@@ -628,7 +628,7 @@ services:
             - '@contao.security.authentication_success_handler'
             - '@contao.security.authentication_failure_handler'
             - []
-            - '@scheb_two_factor.token_factory'
+            - ~ # Backwards compatibility
```

Comment:
There is no need to keep the argument, because event listeners are explicitly excluded from out BC promise: https://docs.contao.org/dev/internals/bc-promise/

Reply:
I'll remove it then 👍

Reply:
Fixed in 9ae780b55.